dbschema/progress-report-workflow-transition.esdl

```diff
@@ -0,0 +1,28 @@
+module default {
```

Comment:
I think this is fine as a separate `esdl`, but I would rename to just `workflow-transition.esdl` and have it inside of the `ProgressReport` module.